### ISSUE TITLE
Debian 8.2 install: lsb_release is missing

### ISSUE THREAD

**Issue by priit** [benhoskings/babushka#332]
Somehow debian 8.2 where missing lsb_release and initial (sh -c "`curl https://babushka.me/up`") babushka install failed with error: 

```
/root/.babushka/bootstrap/lib/babushka/popen.rb:21:in `exec': No such file or directory - lsb_release (Errno::ENOENT)
```

I followed: http://serverfault.com/questions/476485/do-some-debian-builds-not-have-lsb-release where it was suggested to install lsb-release package manually.

If you need help, I could investigate how to permanently create a patch to babushka to fix it, please let me know.

**Comment by schnittchen** [comment]
I can confirm this issue.

**Comment by victorklos** [comment]
This is caused by the fix for #329 btw

**Comment by benhoskings** [comment]
Closing this for now; if it's still a problem on debian patches are welcome.
